src/main/java/at/hannibal2/skyhanni/features/garden/GardenNextJacobContest.kt

```diff
@@ -185,12 +186,7 @@ object GardenNextJacobContest {
     @SubscribeEvent
     fun onInventoryOpen(event: InventoryFullyOpenedEvent) {
         if (!config.display) return
-
-        val backItem = event.inventoryItems[48] ?: return
-        if (backItem.name != "§aGo Back") return
-        val lore = backItem.getLore()
-        if (lore.size != 1) return
-        if (lore[0] != "§7To Calendar and Events") return
```

Comment:
in what inventory does this produce a false positive?

Reply:
Calendar rewards and new chocolate factory on alpha

Reply:
alright!